### ISSUE TITLE
Unable to generate uncensored nudity.

### ISSUE THREAD

**Issue by xxxwangfirexxx** [borisdayma/dalle-mini#239]
No matter what prompts I throw at dalle-mini, I am unable to generate female breasts with nipples, every attempt comes out smooth and lacking nipples. Is there some way to disable the blatant censorship in dalle?

**Comment by Devarite** [comment]
Lmao, no, that is one of the things we do NOT want ai to do.

Also the ai is given a bunch of images to base the output on, and nudity was chosen not to be used for the ai.

**Comment by richaardvark** [comment]
I just wanted to see a picture of Queen Elizabeth II giving birth to a corgi in the self-checkout lane at a Super Walmart... 😢😞

**Comment by vbukovets** [comment]
Fuck censorship, someone has to make an uncensored AI

**Comment by scpedicini** [comment]
I'm far from a prudish person, but one could make the case that generative pornography is far less exploitative than actual pornography since it involves no real people.

I could see a scenario in the future where DALL-E is able to generate video (sequences of contextually consistent pictures) that is pornographic in nature resulting in an overall decrease in the consumption of actual pornography. 

Also "I just want to make out with my monroebot".

**Comment by TakuSmash** [comment]
As controversial an opinion as yours tends to be seen as it's something I've felt similarly about, If these people can just get what they need from AI and don't go after real people anymore that could be seen as a good thing. I'll take all the downvotes you're gonna send my way

**Comment by senseisimple** [comment]
censorship isn't even a concept that's addressed anywhere in the source of this project and others like it...

If you want it, the world is your oyster! Run the project yourself and [train](https://github.com/borisdayma/dalle-mini#training-of-dalle-mini) it with everything you find on some pic site like faphub where all you need are a bounty of images that have some reasonably accurate caption or description... 

the beauty of open source is you can bend it to your will if you're willing to put in the time or cash to have someone else do what you want. I'm sure there's a certain group of people who would praise you for sharing the resulting trained model

**Comment by philiparvidsson** [comment]
Explicit nudity may or may not be censored intentionally in projects like this in the future. I think the implications are enormous either way. You don't need explicit nudity to generate illegal material, and you don't need to use explicit language to express things that would be obviously-illegal.

I think one way of counteracting it would be a second model to determine the impropriety of the generated image and using that do decider whether to allow the result or not, but even then there would be a huge gray zone for obvious reasons.

Beyond that, I think it would require an unfruitful amount of work to implement, and as pointed out already - from a sort-of libertarian perspective - I don't see how preventing people from generating images from the fantasy helps anyone. Au contraire I would think it increases the risk for frustration and acting in the real world, which definitely can cause harm.

The entire concept that is being brought forth right now, I think, implicates that we are moving into a society where people may live their own reality, because reality is becoming less and less shared between individuals. This will surely be a painful phase for society, but I think from where we are right now, we can choose to either accept that or basically criminalize running any kind of AI on any system whatsoever.

From a psychological and maybe even societal standpoint, we might actually be better off stopping everything AI related right now (and this has very little to do with the controversies we are discussing in this thread), but I don't think that is realistic.

TL;DR: Expect to be generating any kind of content you want within a couple of years.

**Comment by CyrilCommando** [comment]
> Lmao, no, that is one of the things we do NOT want ai to do.
> 
> Also the ai is given a bunch of images to base the output on, and nudity was chosen not to be used for that.

Why are you such prudes? For the tech community, you're sure as hell not progressive.

**Comment by SolomidHero** [comment]
> Run the project yourself and [train](https://github.com/borisdayma/dalle-mini#training-of-dalle-mini) it with everything you find on some pic site like faphub

I am not a man of culture, is there some good collection to use? No specific genre
